.travis.yml

```diff
@@ -1,8 +1,14 @@
 language: php
 php:
     - '5.6'
```

Comment:
yeah... guess there's absolutely no way to support both? be a shame to completely drop 5.6